cardinal/server/health.go

```diff
@@ -0,0 +1,22 @@
+package server
+
+import (
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/runtime/middleware/untyped"
+)
+
+type HealthResponse struct {
+	IsServerRunning   bool `json:"is_server_running"`
+	IsGameLoopRunning bool `json:"is_game_loop_running"`
+}
+
+func (handler *Handler) registerHealthHandlerSwagger(api *untyped.API) error {
+	healthHandler := runtime.OperationHandlerFunc(func(params interface{}) (interface{}, error) {
+		res := HealthResponse{
+			true, //see http://ismycomputeron.com/
+			handler.w.IsGameLoopRunning()}
```

Comment:
An alternative pattern here would be to return a 503 if handler.w.IsGameLoopRunning() is false. In this case the client wouldn't have to parse the response body to know the status of the health check.

I'm fine with either pattern.

Reply:
I think it's quite likely that this endpoint will have more information with it related to the status of world-engine. Actually health/ might not be a good name as it's too specific. Will change it when I add more stuff.